### PR TITLE
Drop python 2.7/3.5. Updated readme and test.py examples

### PR DESCRIPTION
**Pull request by steve-gombos**
- Removed python 2.7 and python 3.5.  Min python 3.6+.

- Updated the README (#188 and #190)

- Updated test.py examples for the data refactor that @balloob had previously done. (#186 and #189)

### CODE REVIEW

**Comment by balloob** [comment]
Let's drop everything but Python 3.6+

**Comment by steve-gombos** [comment]
Sounds reasonable to me.  I will get that updated.

File: README.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ Python Ring Door Bell
     :target: https://pypi.python.org/pypi/ring-doorbell
 
 
-Python Ring Door Bell is a library written in Python 2.7/3x
+Python Ring Door Bell is a library written in Python 3x
```
Comment (balloob) [inline README.rst]:
```suggestion
Python Ring Door Bell is a library written in Python 3.6+
```

File: requirements_tests.txt [hunk 2]
```diff
@@ -6,3 +6,4 @@ pytest
 pytest-cov
 requests_mock
 tox
+pathlib
```
Comment (balloob) [inline requirements_tests.txt]: This is not necessary in modern Python, you can drop this.

File: test.py [hunk 3]
```diff
@@ -28,18 +27,16 @@ def main():
     else:
         username = input("Username: ")
         password = input("Password: ")
-        auth = Auth(None, token_updated)
+        auth = Auth("HomeAssistant/0.105.0dev0", None, token_updated)
```
Comment (balloob) [inline test.py]: Please do not identify as Home Assistant. I don't want people to copy the test example and take this over. 

```suggestion
        auth = Auth("MyProject/1.0", None, token_updated)
```

File: requirements_tests.txt (hunk 2 above)
Comment (steve-gombos) [reply requirements_tests.txt]: This also true for requirements.txt?

Comment (balloob) [reply requirements_tests.txt]: I am talking about this one line. Not the whole file. That one is still necessary.

Comment (steve-gombos) [reply requirements_tests.txt]: Long day, I'll get that reverted here shortly.

**Comment by galadril** [comment]
Should ringcli.py also be changed to the new auth/methods?

**Comment by steve-gombos** [comment]
@galadril Didn't even realize this file was there. I'll look at it in a couple hours. Thanks.

**Pull request merged**